src/Main.js

```diff
@@ -1,19 +1,61 @@
 import React from 'react';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import HornedBeast from './HornedBeast.js';
 class Main extends React.Component {
   render() {
+    // let temp = this.props.imageURLs.length;
     return (
-      <>
-        <img src= {this.props.imageURLs[0].image_url} alt={'coming soon'}/>
-        <img src= {this.props.imageURLs[1].image_url} alt={'coming soon'}/>
-        <img src= {this.props.imageURLs[2].image_url} alt={'coming soon'}/>
-        <img src= {this.props.imageURLs[3].image_url} alt={'coming soon'}/>
-        <img src= {this.props.imageURLs[4].image_url} alt={'coming soon'}/>
-        <img src= {this.props.imageURLs[5].image_url} alt={'coming soon'}/>
-        <img src= {this.props.imageURLs[6].image_url} alt={'coming soon'}/>
-        <img src= {this.props.imageURLs[7].image_url} alt={'coming soon'}/>
-      </>
-    )
+      // this.props.imageURLs.forEach ( function(item,i) {
+      <Container className='main'>
+        <Row>
+          <Col>
+            <HornedBeast image_url={this.props.imageURLs[0].image_url}/>
+          </Col>
+          <Col>
+            <HornedBeast image_url={this.props.imageURLs[1].image_url}/>
+          </Col>
+          <Col>
+            <HornedBeast image_url={this.props.imageURLs[2].image_url}/>
+          </Col>
+          <Col>
+            <HornedBeast image_url={this.props.imageURLs[3].image_url}/>
+          </Col>
+        </Row>
+
+        <Row>
+          <Col>
+            <HornedBeast image_url={this.props.imageURLs[4].image_url}/>
+          </Col>
+          <Col>
+            <HornedBeast image_url={this.props.imageURLs[5].image_url}/> 
+          </Col>
+          <Col>
+            <HornedBeast image_url={this.props.imageURLs[6].image_url}/>
+          </Col>
+          <Col>
+            <HornedBeast image_url={this.props.imageURLs[7].image_url}/> 
+          </Col>
+        </Row>
+
+        <Row>
+          <Col>
+        <HornedBeast image_url={this.props.imageURLs[8].image_url}/> 
+          </Col>
+          <Col>
+        <HornedBeast image_url={this.props.imageURLs[9].image_url}/>
+          </Col>
+          <Col>
+        <HornedBeast image_url={this.props.imageURLs[10].image_url}/>
+          </Col>
+          <Col>
+        <HornedBeast image_url={this.props.imageURLs[11].image_url}/>
+          </Col>
+        </Row>
+      </Container>
+    );
```

Comment:
So hard coding al the data in is still a huge problem for your application. Imagine if we updated the JSON file to contain 1000 beasts. There's a couple of simple tricks you can use to loop through the data and create components automatically so that no hard coding is required. Array.map() is a popular method I've seen people use. So think about creating a loop in your Main component which would go through the entire JSON file and for each piece of data, it creates a HornedBeast component.